package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "sep24-reference-ui",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "lint-staged": {
+    "src/**/*.ts?(x)": [
+      "eslint --fix --max-warnings 0"
+    ]
+  },
+  "dependencies": {
```

Comment:
Do we need all this dependencies?

Reply:
Yes, unfortunately, we do. This is fewer dependencies than we used to have. :sweat_smile: